openlibrary/templates/lib/history.html

```diff
@@ -52,12 +52,12 @@ <h2 class="collapse inline">
                 <div class="hidden">
                     <div class="coverFloat" id="wikicode">
                         <div class="coverFloatHead">
-                          <h2>Wikipedia citation</h2>
-                          <a class="dialog--close">&times;<span class="shift">Close</span></a>
+                          <h2>_(Wikipedia citation)</h2>
```

Comment:
Syntax here should be `$_('Wikipedia citation')`, but just reverting to keep this PR focused.
```suggestion
                          <h2>Wikipedia citation</h2>
```